<?xml version="1.0" encoding="UTF-8"?><div xmlns:mml="http://www.w3.org/1998/Math/MathML" id="html-body">

<div id="article-level-0-front-and-body">
<ul xmlns:xs="http://www.w3.org/2001/XMLSchema" class="flat fm article-header-metadata"><li></li><li>ISSN: <span class="issn">1466-5026</span>, Online ISSN: <span class="oissn">1466-5034</span></li><li>DOI: <span class="doi">http://dx.doi.org/10.1099/ijsem.0.002020</span></li><li>Volume <span class="volume">67</span>, Issue <span class="issue">5</span>, pages <span class="first-page">1235</span>-<span class="last-page">1240</span></li><li><span class="copyright">© 2017 IUMS</span> </li></ul><div xmlns:xs="http://www.w3.org/2001/XMLSchema" class="articleSection article-title-and-authors"><div class="articleSection"><div class="sectionDivider"><div class="tl-main-part title"><a name="top-1" id="top-1">
                                 
                            </a></div><div class="menuButton">Go to section...</div><div class="clearer"> </div></div><div class="dropDownMenu"><ul><li><a class="menuLink top-section-link" href="#top-1">TOP</a></li><li><a class="menuLink abstract-section-link" href="#abstract-1">ABSTRACT</a></li><li><a class="menuLink introduction-section-link" href="#introduction-1">INTRODUCTION</a></li><li><a class="menuLink body-section-link" href="#s1-18">Description of <span class="jp-italic">Dyella lipolytica</span> sp. nov.</a></li><li><a class="menuLink ack-section-link" href="#ack1">Funding information</a></li><li><a class="menuLink ack-section-link" href="#ack2">Conflicts of interest</a></li><li><a class="menuLink ref-section-link" href="#references-1">References</a></li></ul></div></div><h1><span class="article-title"><span class="jp-italic">Dyella lipolytica</span> sp. nov., a lipolytic bacterium isolated from lower subtropical forest soil</span></h1><ul class="flat authors">Liang Tang<span class="au-label"></span>†; Mei-hong Chen<span class="au-label"></span>†; Xi-chen Nie<span class="au-label"></span>; Meng-ran Ma<span class="au-label"></span>; Li-hong Qiu<span class="au-label"></span></ul><ul class="flat affiliations">State Key Laboratory of Biocontrol, School of Life Sciences, Sun Yat-sen University, Guangzhou 510275, PR China</ul><div class="correspondence">*Correspondence: Li-hong Qiu, <a href="mailto:qiulh@mail.sysu.edu.cn">qiulh@mail.sysu.edu.cn</a></div><span class="author-footnote" id="FN1"><span class="au-label">†</span><p xmlns:f="http://www.example.org/functions"><span class="fn-label"><span class="au-label">†</span></span>These authors contributed equally to this work.</p></span><span class="author-footnote" id="FN2"><p xmlns:f="http://www.example.org/functions"><span class="fn-label">FN2</span>The GenBank/EMBL/DDBJ accession numbers for the 16S rRNA gene, <span class="jp-italic">gyrB</span>, <span class="jp-italic">lepA</span> and <span class="jp-italic">recA</span> sequences of strain DHOB07<sup>T</sup> are <a target="xrefwindow" href="http://www.ncbi.nlm.nih.gov/nuccore?term=KX430827">KX430827</a>, <a target="xrefwindow" href="http://www.ncbi.nlm.nih.gov/nuccore?term=KX507348">KX507348</a>, <a target="xrefwindow" href="http://www.ncbi.nlm.nih.gov/nuccore?term=KX531237">KX531237</a> and <a target="xrefwindow" href="http://www.ncbi.nlm.nih.gov/nuccore?term=KX531250">KX531250</a>, respectively. The GenBank/EMBL/DDBJ accession numbers for the <span class="jp-italic">gyrB</span>, <span class="jp-italic">lepA</span> and <span class="jp-italic">recA</span> sequences of other strains determined in this study are indicated in Table S1</p></span><span class="author-footnote" id="FN3"><p xmlns:f="http://www.example.org/functions"><span class="fn-label">FN3</span>Three supplementary figures and one supplementary table are available with the online Supplementary Material.</p></span><div class="clearer"> </div></div><div xmlns:xs="http://www.w3.org/2001/XMLSchema" class="articleSection article-abstract"><div class="articleSection"><div class="sectionDivider"><div class="tl-main-part title"><a name="abstract-1" id="abstract-1">ABSTRACT</a></div><div class="menuButton">Go to section...</div><div class="clearer"> </div></div><div class="dropDownMenu"><ul><li><a class="menuLink top-section-link" href="#top-1">TOP</a></li><li><a class="menuLink abstract-section-link" href="#abstract-1">ABSTRACT</a></li><li><a class="menuLink introduction-section-link" href="#introduction-1">INTRODUCTION</a></li><li><a class="menuLink body-section-link" href="#s1-18">Description of <span class="jp-italic">Dyella lipolytica</span> sp. nov.</a></li><li><a class="menuLink ack-section-link" href="#ack1">Funding information</a></li><li><a class="menuLink ack-section-link" href="#ack2">Conflicts of interest</a></li><li><a class="menuLink ref-section-link" href="#references-1">References</a></li></ul></div></div><p>A Gram-stain-negative, aerobic, yellow-pigmented, non-spore-forming, non-motile, rod-shaped bacterium, designated strain DHOB07<sup xmlns:f="http://www.example.org/functions">T</sup>, was isolated from a soil sample collected from the lower subtropical forest of the Dinghushan Biosphere Reserve, Guangdong Province, PR China (23° 10′ N 112° 31′ E). Strain DHOB07<sup xmlns:f="http://www.example.org/functions">T</sup> grew at 10–37 °C, pH 4–7 and 0–0.5 % (w/v) NaCl, with an optimum at 28 °C, pH 5–5.5 and 0% (w/v) NaCl on R2A medium. Phylogenetic analyses based on 16S rRNA gene sequences showed that the strain formed a clade with <span class="jp-italic"><a target="xrefwindow" href="https://doi.org/10.1601/nm.27548" title="Dyella jejuensis - Click to open Names for Life widget" rel="namesforlife-name" class="namesforlife">Dyella jejuensis</a></span> JP1<sup xmlns:f="http://www.example.org/functions">T</sup>, <span class="jp-italic"><a target="xrefwindow" href="https://doi.org/10.1601/nm.29736" title="Dyella nitratireducens - Click to open Names for Life widget" rel="namesforlife-name" class="namesforlife">Dyella nitratireducens</a></span> DHG59<sup xmlns:f="http://www.example.org/functions">T</sup>, <span class="jp-italic"><a target="xrefwindow" href="https://doi.org/10.1601/nm.9534" title="Dyella koreensis - Click to open Names for Life widget" rel="namesforlife-name" class="namesforlife">Dyella koreensis</a></span> BB4<sup xmlns:f="http://www.example.org/functions">T</sup>, <span class="jp-italic"><a target="xrefwindow" href="https://doi.org/10.1601/nm.14370" title="Dyella marensis - Click to open Names for Life widget" rel="namesforlife-name" class="namesforlife">Dyella marensis</a></span> CS5-B2<sup xmlns:f="http://www.example.org/functions">T</sup>and <span class="jp-italic"><a target="xrefwindow" href="https://doi.org/10.1601/nm.14496" title="Dyellasoli - Click to open Names for Life widget" rel="namesforlife-name" class="namesforlife">Dyellasoli</a></span> JS12-10<sup xmlns:f="http://www.example.org/functions">T</sup>, with sequence similarities of 98.9, 98.0, 97.9, 97.9 and 97.8 %, respectively. Multilocus sequence analysis based on the concatenated sequences of partial housekeeping genes <span class="jp-italic">gyrB</span>, <span class="jp-italic">lepA</span> and <span class="jp-italic">recA</span> confirmed that strain DHOB07<sup xmlns:f="http://www.example.org/functions">T</sup> belongs to thegenus <span class="jp-italic"><a target="xrefwindow" href="https://doi.org/10.1601/nm.9359">Dyella</a></span> but is distinct from all currently known species of the genus <span class="jp-italic"><a target="xrefwindow" href="https://doi.org/10.1601/nm.9359">Dyella</a></span>. The G<span class="jp-italic">+</span>C content of the genomic DNA was 58.2 mol%. The DNA–DNA relatedness value between strain DHOB07<sup xmlns:f="http://www.example.org/functions">T</sup> and <span class="jp-italic"><a target="xrefwindow" href="https://doi.org/10.1601/nm.27548" title="D. jejuensis - Click to open Names for Life widget" rel="namesforlife-name" class="namesforlife">D. jejuensis</a></span> JP1<sup xmlns:f="http://www.example.org/functions">T</sup> was 41.8 %. Iso-C<span class="jp-sub">16 : 0</span>, iso-C<span class="jp-sub">15 : 0</span> and iso-C<span class="jp-sub">17 : 1</span>ω9<span class="jp-italic">c</span> were the major fatty acids, and ubiquinone-8 was the only respiratory quinone detected, all of which supported the affiliation of strain DHOB07<sup xmlns:f="http://www.example.org/functions">T</sup> to the genus <span class="jp-italic"><a target="xrefwindow" href="https://doi.org/10.1601/nm.9359" title="Dyella - Click to open Names for Life widget" rel="namesforlife-name" class="namesforlife">Dyella</a></span>. On the basis of the polyphasic characterization results presented above, strain DHOB07<sup xmlns:f="http://www.example.org/functions">T</sup> represents a novel species of the genus <span class="jp-italic"><a target="xrefwindow" href="https://doi.org/10.1601/nm.9359" title="Dyella - Click to open Names for Life widget" rel="namesforlife-name" class="namesforlife">Dyella</a></span>, for which the name <span class="jp-italic">Dyella lipolytica</span> sp. nov. is proposed. The type strain is DHOB07<sup xmlns:f="http://www.example.org/functions">T</sup> (=NBRC 111473<sup xmlns:f="http://www.example.org/functions">T</sup>=KCTC 52132<sup xmlns:f="http://www.example.org/functions">T</sup>).</p>
<span class="tl-lowest-section">Keywords</span><ul class="flat keywords"><li><span class="jp-italic">Dyella lipolytica</span> sp. nov</li></ul><div class="clearer"> </div></div><span xmlns:xs="http://www.w3.org/2001/XMLSchema" class="tl-lowest-section">Author Notes</span><div xmlns:xs="http://www.w3.org/2001/XMLSchema" class="author-footnote-below-abstract" id="FN1">†These authors contributed equally to this work.</div><div xmlns:xs="http://www.w3.org/2001/XMLSchema" class="author-footnote-below-abstract" id="FN2">The GenBank/EMBL/DDBJ accession numbers for the 16S rRNA gene, gyrB, lepA and recA sequences of strain DHOB07T are <a target="xrefwindow" href="http://www.ncbi.nlm.nih.gov/nuccore?term=KX430827">KX430827</a>, <a target="xrefwindow" href="http://www.ncbi.nlm.nih.gov/nuccore?term=KX507348">KX507348</a>, <a target="xrefwindow" href="http://www.ncbi.nlm.nih.gov/nuccore?term=KX531237">KX531237</a> and <a target="xrefwindow" href="http://www.ncbi.nlm.nih.gov/nuccore?term=KX531250">KX531250</a>, respectively. The GenBank/EMBL/DDBJ accession numbers for the gyrB, lepA and recA sequences of other strains determined in this study are indicated in Table S1</div><div xmlns:xs="http://www.w3.org/2001/XMLSchema" class="author-footnote-below-abstract" id="FN3">Three supplementary figures and one supplementary table are available with the online Supplementary Material.</div>



<div xmlns:xs="http://www.w3.org/2001/XMLSchema" class="articleSection"><div class="articleSection"><div class="sectionDivider"><div class="tl-main-part title"><a name="introduction-1" id="introduction-1">
                                 
                            </a></div><div class="menuButton">Go to section...</div><div class="clearer"> </div></div><div class="dropDownMenu"><ul><li><a class="menuLink top-section-link" href="#top-1">TOP</a></li><li><a class="menuLink abstract-section-link" href="#abstract-1">ABSTRACT</a></li><li><a class="menuLink introduction-section-link" href="#introduction-1">INTRODUCTION</a></li><li><a class="menuLink body-section-link" href="#s1-18">Description of <span class="jp-italic">Dyella lipolytica</span> sp. nov.</a></li><li><a class="menuLink ack-section-link" href="#ack1">Funding information</a></li><li><a class="menuLink ack-section-link" href="#ack2">Conflicts of interest</a></li><li><a class="menuLink ref-section-link" href="#references-1">References</a></li></ul></div></div><p>The genus <span class="jp-italic"><a target="xrefwindow" href="https://doi.org/10.1601/nm.9359" title="Dyella - Click to open Names for Life widget" rel="namesforlife-name" class="namesforlife">Dyella</a></span> belongs to the family <span class="jp-italic"><a target="xrefwindow" href="https://doi.org/10.1601/nm.2207" title="Xanthomonadaceae - Click to open Names for Life widget" rel="namesforlife-name" class="namesforlife">Xanthomonadaceae</a></span> of the phylum <span class="jp-italic"><a target="xrefwindow" href="https://doi.org/10.1601/nm.808" title="Proteobacteria - Click to open Names for Life widget" rel="namesforlife-name" class="namesforlife">Proteobacteria</a></span>. Since Xie and Yokota [<span class="xref"><a xmlns:f="http://www.example.org/functions" href="#R1">1</a></span>] established the genus <span class="jp-italic"><a target="xrefwindow" href="https://doi.org/10.1601/nm.9359" title="Dyella - Click to open Names for Life widget" rel="namesforlife-name" class="namesforlife">Dyella</a></span> with <span class="jp-italic"><a target="xrefwindow" href="https://doi.org/10.1601/nm.9358" title="D. japonica - Click to open Names for Life widget" rel="namesforlife-name" class="namesforlife">D. japonica</a></span> as the type species in 2005, another 13 species have been recognized: <span class="jp-italic"><a target="xrefwindow" href="https://doi.org/10.1601/nm.9534" title="D. koreensis - Click to open Names for Life widget" rel="namesforlife-name" class="namesforlife">D. koreensis</a></span> [<span class="xref"><a xmlns:f="http://www.example.org/functions" href="#R2">2</a></span>], <span class="jp-italic"><a target="xrefwindow" href="https://doi.org/10.1601/nm.24607" title="D. kyungheensis - Click to open Names for Life widget" rel="namesforlife-name" class="namesforlife">D. kyungheensis</a></span> [<span class="xref"><a xmlns:f="http://www.example.org/functions" href="#R3">3</a></span>], <span class="jp-italic"><a target="xrefwindow" href="https://doi.org/10.1601/nm.14140" title="D. ginsengisoli - Click to open Names for Life widget" rel="namesforlife-name" class="namesforlife">D. ginsengisoli</a></span> [<span class="xref"><a xmlns:f="http://www.example.org/functions" href="#R4">4</a></span>], <span class="jp-italic"><a target="xrefwindow" href="https://doi.org/10.1601/nm.24483" title="D. jiangningensis - Click to open Names for Life widget" rel="namesforlife-name" class="namesforlife">D. jiangningensis</a></span> [<span class="xref"><a xmlns:f="http://www.example.org/functions" href="#R5">5</a></span>], <span class="jp-italic"><a target="xrefwindow" href="https://doi.org/10.1601/nm.14370" title="D. marensis - Click to open Names for Life widget" rel="namesforlife-name" class="namesforlife">D. marensis</a></span> [<span class="xref"><a xmlns:f="http://www.example.org/functions" href="#R6">6</a></span>], <span class="jp-italic"><a target="xrefwindow" href="https://doi.org/10.1601/nm.14496" title="D. soli - Click to open Names for Life widget" rel="namesforlife-name" class="namesforlife">D. soli</a></span>, <span class="jp-italic"><a target="xrefwindow" href="https://doi.org/10.1601/nm.14497" title="D. terrae - Click to open Names for Life widget" rel="namesforlife-name" class="namesforlife">D. terrae</a></span> [<span class="xref"><a xmlns:f="http://www.example.org/functions" href="#R7">7</a></span>], <span class="jp-italic"><a target="xrefwindow" href="https://doi.org/10.1601/nm.21165" title="D. thiooxydans - Click to open Names for Life widget" rel="namesforlife-name" class="namesforlife">D. thiooxydans</a></span> [<span class="xref"><a xmlns:f="http://www.example.org/functions" href="#R8">8</a></span>], <span class="jp-italic"><a target="xrefwindow" href="https://doi.org/10.1601/nm.27548" title="D. jejuensis - Click to open Names for Life widget" rel="namesforlife-name" class="namesforlife">D. jejuensis</a></span> [<span class="xref"><a xmlns:f="http://www.example.org/functions" href="#R9">9</a></span>], <span class="jp-italic"><a target="xrefwindow" href="https://doi.org/10.1601/nm.29165" title="D. humi - Click to open Names for Life widget" rel="namesforlife-name" class="namesforlife">D. humi</a></span> [<span class="xref"><a xmlns:f="http://www.example.org/functions" href="#R10">10</a></span>], <span class="jp-italic"><a target="xrefwindow" href="https://doi.org/10.1601/nm.29734" title="D. acidisoli - Click to open Names for Life widget" rel="namesforlife-name" class="namesforlife">D. acidisoli</a></span>, <span class="jp-italic"><a target="xrefwindow" href="https://doi.org/10.1601/nm.29735" title="D. flagellata - Click to open Names for Life widget" rel="namesforlife-name" class="namesforlife">D. flagellata</a></span> and <span class="jp-italic"><a target="xrefwindow" href="https://doi.org/10.1601/nm.29736" title="D. nitratireducens - Click to open Names for Life widget" rel="namesforlife-name" class="namesforlife">D. nitratireducens</a></span>[<span class="xref"><a xmlns:f="http://www.example.org/functions" href="#R11">11</a></span>]. These bacteria were isolated mainly from soil and are Gram-negative, aerobic, rod-shaped and have ubiquinone-8 as the major respiratory quinone. In the present study, we describe a novel species of the genus <span class="jp-italic"><a target="xrefwindow" href="https://doi.org/10.1601/nm.9359" title="Dyella - Click to open Names for Life widget" rel="namesforlife-name" class="namesforlife">Dyella</a></span> based on a polyphasic study on a locally isolated bacterial strain, designated DHOB07<sup xmlns:f="http://www.example.org/functions">T</sup>.</p>
<p>Strain DHOB07<sup xmlns:f="http://www.example.org/functions">T</sup> was isolated from a soil sample collected from the lower subtropical forest of Dinghushan Biosphere Reserve, Guangdong Province, PR China (23° 10′ N 112° 31′ E). The soil sample was acidic lateritic red earth with a pH of 4.0–5.0. For the isolation, the soil sample was suspended in PBS and the resulting suspensions were serially diluted with the same buffer. Aliquots of the diluted suspensions were spread on MM1F medium (MgSO<span class="jp-sub">4</span>.7H<span class="jp-sub">2</span>O 0.04 g, fructose 0.5 g, CaCl<span class="jp-sub">2</span>.2H<span class="jp-sub">2</span>O 0.02 g, yeast 0.05 g, agar 15 g, in 1 litre distilled water, pH 5.5) and the plates were incubated at 28 °C for 1 week. Once a single colony was obtained, it was purified by sub-culturing three times under the same conditions. Strain DHOB07<sup xmlns:f="http://www.example.org/functions">T</sup> was isolated in this way. The isolate was maintained in a 25 % (v/v) glycerol suspension in a refrigerator at −80 °C. The growth of strain DHOB07<sup xmlns:f="http://www.example.org/functions">T</sup> was tested on trypticase soy agar (TSA; HKM), R2A agar (HKM), nutrient agar (NA; HKM), Luria–Bertani agar (LB; HKM), and MacConkey agar (HKM). No sign of growth of DHOB07<sup xmlns:f="http://www.example.org/functions">T</sup> was observed on TSA, NA, LB and MacConkey agar medium, and the best growth was achieved on R2A agar at 28 °C for 2 days. Strain DHOB07<sup xmlns:f="http://www.example.org/functions">T</sup> and the type strain of its closest described species <span class="jp-italic"><a target="xrefwindow" href="https://doi.org/10.1601/nm.27548" title="D. jejuensis - Click to open Names for Life widget" rel="namesforlife-name" class="namesforlife">D. jejuensis</a></span> JP1<sup xmlns:f="http://www.example.org/functions">T</sup>, which was used as the reference strain in this study, were routinely grown aerobically on R2A agar at 28 °C for 2 days except where otherwise stated.</p>
<p>The Gram-staining reaction was determined using the standard staining protocol [<span class="xref"><a xmlns:f="http://www.example.org/functions" href="#R12">12</a></span>]. The cell morphology of strain DHOB07<sup xmlns:f="http://www.example.org/functions">T</sup> was examined using an optical microscope (×1000 magnification, Nikon) with cells grown on both liquid and solid R2A media. Cell motility was studied by observing the development of turbidity throughout a tube of semi-solid (0.3 % agar) R2A agar. Growth at different temperatures (20, 25, 28, 33, 37 and 42 °C) and tolerance to various concentrations of NaCl (0, 0.5, 1, 1.5, 2, 2.5, 3, 3.5, 4, 4.5 and 5.0 % w/v) was determined using R2A medium. The pH range for growth was tested from pH 3.0 to 10.0 (in increments of 0.5 units) using the buffer system described by Xu <span class="jp-italic">et al.</span> [<span class="xref"><a xmlns:f="http://www.example.org/functions" href="#R13">13</a></span>]. The development of colonies was observed on R2A solid media at 4 and 10 °C for 2 weeks.</p>
<p>Cells of strain DHOB07<sup xmlns:f="http://www.example.org/functions">T</sup> were aerobic, non-spore-forming, non-motile and rod-shaped (0.3–0.6 µm wide and 1.0–2.5 µm long). Colonies of strain DHOB07<sup xmlns:f="http://www.example.org/functions">T</sup> grown on R2A agar for 2 days were smooth, circular (1.2–1.7 mm in diameter), yellow-pigmented, semi-transparent and convex with clear edges. The strain was Gram-staining-negative. On R2A medium, strain DHOB07<sup xmlns:f="http://www.example.org/functions">T</sup> grew at 10–37 °C, pH 4–7 and with 0–0.5 % (w/v) NaCl, with optimum growth at 28 °C, pH 5–5.5 and with 0 % (w/v) NaCl.</p>
<p>Catalase activity was detected by checking bubble formation in 3 % (v/v) hydrogen peroxide, and oxidase activity was determined by using 1 % (w/v) tetramethyl-<span class="jp-italic">p</span>-phenylenediamine oxidase reagent. Enzyme activities and other physiological and biochemical characteristics were determined with API ZYM, API 20 NE and API 50 CHB/E galleries (bioMérieux) according to the manufacturer’s instructions. Tests for the hydrolysis of casein, starch [<span class="xref"><a xmlns:f="http://www.example.org/functions" href="#R14">14</a></span>] and Tween 80 [<span class="xref"><a xmlns:f="http://www.example.org/functions" href="#R15">15</a></span>] were assessed after incubation for 3 days at 28 °C.</p>
<p>Strain DHOB07<sup xmlns:f="http://www.example.org/functions">T</sup> was positive for oxidase activity and Tween 80 hydrolysis, but negative for catalase activity, and starch and casein hydrolysis. Like <span class="jp-italic"><a target="xrefwindow" href="https://doi.org/10.1601/nm.27548" title="D. jejuensis - Click to open Names for Life widget" rel="namesforlife-name" class="namesforlife">D. jejuensis</a></span> JP1<sup xmlns:f="http://www.example.org/functions">T</sup>, <span class="jp-italic"><a target="xrefwindow" href="https://doi.org/10.1601/nm.29735" title="D. flagellata - Click to open Names for Life widget" rel="namesforlife-name" class="namesforlife">D. flagellata</a></span> 4M-K16<sup xmlns:f="http://www.example.org/functions">T</sup> and <span class="jp-italic"><a target="xrefwindow" href="https://doi.org/10.1601/nm.29736" title="D. nitratireducens - Click to open Names for Life widget" rel="namesforlife-name" class="namesforlife">D. nitratireducens</a></span> DHG59<sup xmlns:f="http://www.example.org/functions">T</sup>, strain DHOB07<sup xmlns:f="http://www.example.org/functions">T</sup> had poor capacity to utilize gelatin, lactose, methyl <span class="jp-italic">α</span>-<span class="jp-small">d</span>-glucopyranoside, potassium 2-ketogluconate, salicin and starch. However, erythritol, glycogen and inositol were utilized only by strain DHOB07<sup xmlns:f="http://www.example.org/functions">T</sup>. The detailed physiological and biochemical characteristics that differentiate strain DHOB07<sup xmlns:f="http://www.example.org/functions">T</sup> from its closest phylogenetic relatives are listed in <span xmlns:f="http://www.example.org/functions" class="xref"><a href="#T1">Table 1</a></span>.</p>
<!--splitRids is T1--><!--currentRid is T1--><!--current position is 24--><!--check position is 24--><div class="table-caption-container js-table-caption-container" data-webId="/content/ijsem/10.1099/ijsem.0.002020.T1" id="T1"><h5 class="table-label">Table 1.</h5><p><span>Differential characteristics between strain DHOB07<sup xmlns:f="http://www.example.org/functions">T</sup> and the type strains of related species of the genus <span class="jp-italic"><a target="xrefwindow" href="https://doi.org/10.1601/nm.9359" title="Dyella - Click to open Names for Life widget" rel="namesforlife-name" class="namesforlife">Dyella</a></span></span>
<span>Strains: 1, DHOB07<sup xmlns:f="http://www.example.org/functions">T</sup>; 2, <span class="jp-italic"><a target="xrefwindow" href="https://doi.org/10.1601/nm.27548" title="D. jejuensis - Click to open Names for Life widget" rel="namesforlife-name" class="namesforlife">D. jejuensis</a></span> JP1<sup xmlns:f="http://www.example.org/functions">T</sup>; 3, <span class="jp-italic"><a target="xrefwindow" href="https://doi.org/10.1601/nm.29735" title="D. flagellata - Click to open Names for Life widget" rel="namesforlife-name" class="namesforlife">D. flagellata</a></span> 4M-K16<sup xmlns:f="http://www.example.org/functions">T</sup>; 4, <span class="jp-italic"><a target="xrefwindow" href="https://doi.org/10.1601/nm.29736" title="D. nitratireducens - Click to open Names for Life widget" rel="namesforlife-name" class="namesforlife">D. nitratireducens</a></span> DHG59<sup xmlns:f="http://www.example.org/functions">T</sup>; 5, <span class="jp-italic"><a target="xrefwindow" href="https://doi.org/10.1601/nm.29165" title="D. humi - Click to open Names for Life widget" rel="namesforlife-name" class="namesforlife">D. humi</a></span> DHG40<sup xmlns:f="http://www.example.org/functions">T</sup>; 6, <span class="jp-italic"><a target="xrefwindow" href="https://doi.org/10.1601/nm.9534" title="D. koreensis - Click to open Names for Life widget" rel="namesforlife-name" class="namesforlife">D. koreensis</a></span> BB4<sup xmlns:f="http://www.example.org/functions">T</sup>; 7, <span class="jp-italic"><a target="xrefwindow" href="https://doi.org/10.1601/nm.14370" title="D. marensis - Click to open Names for Life widget" rel="namesforlife-name" class="namesforlife">D. marensis</a></span> CS5-B2<sup xmlns:f="http://www.example.org/functions">T</sup>; 8, <span class="jp-italic"><a target="xrefwindow" href="https://doi.org/10.1601/nm.29734" title="D. acidisoli - Click to open Names for Life widget" rel="namesforlife-name" class="namesforlife">D. acidisoli</a></span> 4M-Z03<sup xmlns:f="http://www.example.org/functions">T</sup>. Data for strains 1 and 2 from this study, data for strains 3–8 from Chen <span class="jp-italic">et al</span>. [<span class="xref"><a xmlns:f="http://www.example.org/functions" href="#R11">11</a></span>]. +, Positive; <span class="jp-small">w</span>, weakly positive; −, negative.</span>
</p></div><p>Antimicrobial susceptibility of strain DHOB07<sup xmlns:f="http://www.example.org/functions">T</sup> was tested using the Kirby Bauer method [<span class="xref"><a xmlns:f="http://www.example.org/functions" href="#R16">16</a></span>]. The following 14 antibiotics were tested: amikacin (30 µg), chloramphenicol (30 µg), ciprofloxacin (5 µg), erythromycin (15 µg), gentamicin (10 µg), kanamycin (30 µg), netilmicin (30 µg), novobiocin (5 µg), penicillin (10 IU), polymyxin (300 U), streptomycin (10 µg), tetracycline (30 µg), tobramycin (10 µg) and vancomycin (30 µg). Results were determined by measuring the zone of inhibition appearing on R2A agar medium after incubation for 2 days at 28 °C. Strain DHOB07<sup xmlns:f="http://www.example.org/functions">T</sup> was resistant to chloramphenicol and penicillin but susceptible to all other antibiotics tested.</p>
<p>Genomic DNA of strain DHOB07<sup xmlns:f="http://www.example.org/functions">T</sup> was extracted with a commercial genomic DNA extraction kit (GeneStar). The almost full-length 16S rRNA gene and the partial <span class="jp-italic">gyrB</span>, <span class="jp-italic">lepA</span> and <span class="jp-italic">recA</span> gene sequences of strain DHOB07<sup xmlns:f="http://www.example.org/functions">T</sup> were amplified and sequenced following the methodology detailed by Chen <span class="jp-italic">et al.</span> [<span class="xref"><a xmlns:f="http://www.example.org/functions" href="#R11">11</a></span>]. The newly determined gene sequences were quality-checked and then submitted to the GenBank database. Phylogenetic dendrograms were generated with three tree-making algorithms: neighbour-joining [<span class="xref"><a xmlns:f="http://www.example.org/functions" href="#R17">17</a></span>], maximum-parsimony [<span class="xref"><a xmlns:f="http://www.example.org/functions" href="#R18">18</a></span>] and maximum-likelihood [<span class="xref"><a xmlns:f="http://www.example.org/functions" href="#R19">19</a></span>] using <span class="jp-small">mega</span> software package version 5.0 [<span class="xref"><a xmlns:f="http://www.example.org/functions" href="#R20">20</a></span>]. Evolutionary distance matrices (distance options according to Kimura’s two-parameter model) were generated as described by Kimura [<span class="xref"><a xmlns:f="http://www.example.org/functions" href="#R21">21</a></span>] and the tree topology was assessed by bootstrap analyses [<span class="xref"><a xmlns:f="http://www.example.org/functions" href="#R22">22</a></span>] based on 1000 replications.</p>
<p>A comparison between the continuous stretch of 16S rRNA gene sequence of strain DHOB07<sup xmlns:f="http://www.example.org/functions">T</sup> and those of other reported strains indicated that strain DHOB07<sup xmlns:f="http://www.example.org/functions">T</sup> belongs to the genus <span class="jp-italic"><a target="xrefwindow" href="https://doi.org/10.1601/nm.9359" title="Dyella - Click to open Names for Life widget" rel="namesforlife-name" class="namesforlife">Dyella</a></span>. The phylogenetic tree reconstructed using the neighbour-joining method (<span xmlns:f="http://www.example.org/functions" class="xref"><a href="#F1">Fig. 1</a></span>) showed that strain DHOB07<sup xmlns:f="http://www.example.org/functions">T</sup> formed a highly supported clade with <span class="jp-italic"><a target="xrefwindow" href="https://doi.org/10.1601/nm.27548" title="D. jejuensis - Click to open Names for Life widget" rel="namesforlife-name" class="namesforlife">D. jejuensis</a></span> JP1<sup xmlns:f="http://www.example.org/functions">T</sup> (98.9 % 16S rRNA gene sequence similarity), <span class="jp-italic"><a target="xrefwindow" href="https://doi.org/10.1601/nm.29736" title="D. nitratireducens - Click to open Names for Life widget" rel="namesforlife-name" class="namesforlife">D. nitratireducens</a></span> DHG59<sup xmlns:f="http://www.example.org/functions">T</sup> (98.0 %), <span class="jp-italic"><a target="xrefwindow" href="https://doi.org/10.1601/nm.9534" title="D. koreensis - Click to open Names for Life widget" rel="namesforlife-name" class="namesforlife">D. koreensis</a></span> BB4<sup xmlns:f="http://www.example.org/functions">T</sup> (97.9 %), <span class="jp-italic"><a target="xrefwindow" href="https://doi.org/10.1601/nm.14370" title="D. marensis - Click to open Names for Life widget" rel="namesforlife-name" class="namesforlife">D. marensis</a></span> CS5-B2<sup xmlns:f="http://www.example.org/functions">T</sup> (97.9 %) and <span class="jp-italic"><a target="xrefwindow" href="https://doi.org/10.1601/nm.14496" title="D. soli - Click to open Names for Life widget" rel="namesforlife-name" class="namesforlife">D. soli</a></span> JS12-10<sup xmlns:f="http://www.example.org/functions">T</sup> (97.8 %) as its phylogenetically closest relatives. This relationship was also supported by the other two tree-making methods maximum-likelihood and maximum-parsimony [Figs S1 and S2 (available in the online Supplementary Material)]. The partial sequences of the housekeeping genes <span class="jp-italic">gyrB</span> (768 bp), <span class="jp-italic">lepA</span> (777 bp) and <span class="jp-italic">recA</span> (809 bp) of strain DHOB07<sup xmlns:f="http://www.example.org/functions">T</sup> were submitted to the GenBank database (Table S1). The neighbour-joining phylogenetic tree (<span xmlns:f="http://www.example.org/functions" class="xref"><a href="#F2">Fig. 2</a></span>) based on the concatenated sequences of the above three housekeeping genes showed the strain DHOB07<sup xmlns:f="http://www.example.org/functions">T</sup>formed a robust cluster with <span class="jp-italic"><a target="xrefwindow" href="https://doi.org/10.1601/nm.27548" title="D. jejuensis - Click to open Names for Life widget" rel="namesforlife-name" class="namesforlife">D. jejuensis</a></span> JP1<sup xmlns:f="http://www.example.org/functions">T</sup> (85.2 % concatenated partial <span class="jp-italic">gyrB</span>, <span class="jp-italic">lepA</span> and <span class="jp-italic">recA housekeeping genes sequencce similarity</span>), <span class="jp-italic"><a target="xrefwindow" href="https://doi.org/10.1601/nm.29736" title="D. nitratireducens - Click to open Names for Life widget" rel="namesforlife-name" class="namesforlife">D. nitratireducens</a></span> DHG59<sup xmlns:f="http://www.example.org/functions">T</sup> (85.9 %), <span class="jp-italic"><a target="xrefwindow" href="https://doi.org/10.1601/nm.29735" title="D. flagellata - Click to open Names for Life widget" rel="namesforlife-name" class="namesforlife">D. flagellata</a></span> 4M-K16<sup xmlns:f="http://www.example.org/functions">T</sup> (85.5 %), <span class="jp-italic"><a target="xrefwindow" href="https://doi.org/10.1601/nm.29734" title="D. acidisoli - Click to open Names for Life widget" rel="namesforlife-name" class="namesforlife">D. acidisoli</a></span> 4M-Z03<sup xmlns:f="http://www.example.org/functions">T</sup> (85.3 %) and <span class="jp-italic"><a target="xrefwindow" href="https://doi.org/10.1601/nm.29165" title="D. humi - Click to open Names for Life widget" rel="namesforlife-name" class="namesforlife">D. humi</a></span> DHG40<sup xmlns:f="http://www.example.org/functions">T</sup> (86.8 %), which clearly suggested that strain DHOB07<sup xmlns:f="http://www.example.org/functions">T</sup> is a member of the genus <span class="jp-italic"><a target="xrefwindow" href="https://doi.org/10.1601/nm.9359" title="Dyella - Click to open Names for Life widget" rel="namesforlife-name" class="namesforlife">Dyella</a></span>. The low sequence similarity values together with high evolutionary distance between strain DHOB07<sup xmlns:f="http://www.example.org/functions">T</sup> and the other established species of the genus showed that it represents a novel species of the genus <span class="jp-italic"><a target="xrefwindow" href="https://doi.org/10.1601/nm.9359" title="Dyella - Click to open Names for Life widget" rel="namesforlife-name" class="namesforlife">Dyella</a></span>.</p>
<!--splitRids is F1--><!--currentRid is F1--><!--current position is 34--><!--check position is 34--><!--splitRids is F2--><!--currentRid is F2--><!--current position is 35--><!--check position is 35--><p xmlns:f="http://www.example.org/functions"><div class="figure html-fulltext-responsive-figure" id="F1"><div class="caption"><span class="captionLabel"><span class="label">Fig. 1.</span></span><span class="captionText"><span>Neighbour-joining tree based on 16S rRNA gene sequence comparison, showing the relationship of strain DHOB07<sup>T</sup> to other species of the genera <span class="jp-italic"><a target="xrefwindow" href="https://doi.org/10.1601/nm.9359" title="Dyella - Click to open Names for Life widget" rel="namesforlife-name" class="namesforlife">Dyella</a></span>, <span class="jp-italic"><a target="xrefwindow" href="https://doi.org/10.1601/nm.2251" title="Rhodanobacter - Click to open Names for Life widget" rel="namesforlife-name" class="namesforlife">Rhodanobacter</a></span> and <span class="jp-italic"><a target="xrefwindow" href="https://doi.org/10.1601/nm.2234" title="Frateuria - Click to open Names for Life widget" rel="namesforlife-name" class="namesforlife">Frateuria</a></span>. Bootstrap values are shown in percentages of 1000 replicates when &gt;50 %. <span class="jp-italic"><a target="xrefwindow" href="https://doi.org/10.1601/nm.2209" title="Xanthomonas campestris - Click to open Names for Life widget" rel="namesforlife-name" class="namesforlife">Xanthomonas campestris</a></span> ATCC 33913<sup>T</sup> was used as an outgroup. Bar, 0.01 substitutions per nucleotide position.</span>
</span></div><div class="image"><a class="media-link" href="/content/ijsem/10.1099/ijsem.0.002020.F1" id="/content/ijsem/10.1099/ijsem.0.002020.F1"><img src="/docserver/ahah/fulltext/ijsem/67/5/ijsem002020-f1_thmb.gif" border="0" alt="Fig. 1."></img><p><span class="figure-duplicate-label">Fig. 1.<br></br></span>Click to view</p></a></div><div class="clearer"> </div></div></p><p xmlns:f="http://www.example.org/functions"><div class="figure html-fulltext-responsive-figure" id="F2"><div class="caption"><span class="captionLabel"><span class="label">Fig. 2.</span></span><span class="captionText"><span>Neighbour-joining tree showing the phylogenetic position of strain DHOB07<sup>T</sup> in the genus <span class="jp-italic"><a target="xrefwindow" href="https://doi.org/10.1601/nm.9359" title="Dyella - Click to open Names for Life widget" rel="namesforlife-name" class="namesforlife">Dyella</a></span>, based on the concatenated partial sequences of three housekeeping genes <span class="jp-italic">gyrB</span>, <span class="jp-italic">lepA</span> and <span class="jp-italic">recA</span>. <span class="jp-italic"><a target="xrefwindow" href="https://doi.org/10.1601/nm.2209" title="Xanthomonas campestris - Click to open Names for Life widget" rel="namesforlife-name" class="namesforlife">Xanthomonas campestris</a></span> ATCC 33913<sup>T</sup> was used as an outgroup. Bootstrap values &gt;50 % based on 1000 replicates are listed as percentages at branching points. Bar, 0.02 substitutions per nucleotide position.</span>
</span></div><div class="image"><a class="media-link" href="/content/ijsem/10.1099/ijsem.0.002020.F2" id="/content/ijsem/10.1099/ijsem.0.002020.F2"><img src="/docserver/ahah/fulltext/ijsem/67/5/ijsem002020-f2_thmb.gif" border="0" alt="Fig. 2."></img><p><span class="figure-duplicate-label">Fig. 2.<br></br></span>Click to view</p></a></div><div class="clearer"> </div></div></p><p>The DNA G+C content of strain DHOB07<sup xmlns:f="http://www.example.org/functions">T</sup> was determined with reversed-phase HPLC [<span class="xref"><a xmlns:f="http://www.example.org/functions" href="#R23">23</a></span>] using genomic DNA extracted as described by Marmur and Doty [<span class="xref"><a xmlns:f="http://www.example.org/functions" href="#R24">24</a></span>]. DNA–DNA hybridization experiments were conducted in triplicate using a Lambda 35 UV/VIS spectrometer equipped with a temperature program controller (PerkinElmer), as described by De Ley <span class="jp-italic">et al.</span> [<span class="xref"><a xmlns:f="http://www.example.org/functions" href="#R25">25</a></span>]. The DNA G+C content of strains DHOB07<sup xmlns:f="http://www.example.org/functions">T</sup> was 58.2 mol%, within the range of the genus <span class="jp-italic"><a target="xrefwindow" href="https://doi.org/10.1601/nm.9359" title="Dyella - Click to open Names for Life widget" rel="namesforlife-name" class="namesforlife">Dyella</a></span>. The DNA–DNA relatedness value between strain DHOB07<sup xmlns:f="http://www.example.org/functions">T</sup> and <span class="jp-italic"><a target="xrefwindow" href="https://doi.org/10.1601/nm.27548" title="D. jejuensis - Click to open Names for Life widget" rel="namesforlife-name" class="namesforlife">D. jejuensis</a></span> JP1<sup xmlns:f="http://www.example.org/functions">T</sup> was 41.8 %, which is significantly below the threshold of 70 % proposed for species discrimination by Wayne <span class="jp-italic">et al.</span> [<span class="xref"><a xmlns:f="http://www.example.org/functions" href="#R26">26</a></span>].</p>
<p>Fatty acid methyl esters of strain DHOB07<sup xmlns:f="http://www.example.org/functions">T</sup> and the type strains of selected species of the genus <span class="jp-italic"><a target="xrefwindow" href="https://doi.org/10.1601/nm.9359" title="Dyella - Click to open Names for Life widget" rel="namesforlife-name" class="namesforlife">Dyella</a></span> were obtained from 40 mg cells grown on R2A agar for 2 days at 28 °C using the method of Kuykendall <span class="jp-italic">et al.</span> [<span class="xref"><a xmlns:f="http://www.example.org/functions" href="#R27">27</a></span>]. The mixtures of fatty acid methyl esters were separated by GC and determined using the Sherlock Microbial Identification System (MIS) (MIDI). Respiratory quinones were extracted and identified according to the methods of Minnikin <span class="jp-italic">et al.</span> [<span class="xref"><a xmlns:f="http://www.example.org/functions" href="#R28">28</a></span>] and separated by HPLC [<span class="xref"><a xmlns:f="http://www.example.org/functions" href="#R29">29</a></span>]. Polar lipids were extracted and analysed by two-dimensional TLC according to Minnikin <span class="jp-italic">et al.</span> [<span class="xref"><a xmlns:f="http://www.example.org/functions" href="#R28">28</a></span>].</p>
<p>The major fatty acids of strain DHOB07<sup xmlns:f="http://www.example.org/functions">T</sup> were iso-C<span class="jp-sub">17 : 1</span><span class="jp-italic"> ω</span>9<span class="jp-italic">c</span> (20.4 %), iso-C<span class="jp-sub">16 : 0</span> (19.2 %), iso-C<span class="jp-sub">15 : 0</span> (14.3 %), and summed feature 3 (C<span class="jp-sub">16 : 1</span>ω6<span class="jp-italic">c</span> and/or C<span class="jp-sub">16 : 1</span>ω7<span class="jp-italic">c</span>, 7.6 %), which are the characteristics of the members of the genus <span class="jp-italic"><a target="xrefwindow" href="https://doi.org/10.1601/nm.9359" title="Dyella - Click to open Names for Life widget" rel="namesforlife-name" class="namesforlife">Dyella</a></span> (<span xmlns:f="http://www.example.org/functions" class="xref"><a href="#T2">Table 2</a></span>). The major respiratory quinone of strain DHOB07<sup xmlns:f="http://www.example.org/functions">T</sup> was ubiquinone-8 (Q-8), consistent with all other members of the genus <span class="jp-italic"><a target="xrefwindow" href="https://doi.org/10.1601/nm.9359" title="Dyella - Click to open Names for Life widget" rel="namesforlife-name" class="namesforlife">Dyella</a></span>. The major polar lipids of strain DHOB07<sup xmlns:f="http://www.example.org/functions">T</sup> were phosphatidylethanolamine, phosphatidylglycerol, diphosphatidylglycerol, phosphatidylmethylethanolamine, and several unidentified aminophospholipids (APL 1–2), unidentified aminolipids (AL), phospholipids (PL 1–2) and lipids (Fig. S3). This polar lipid profile was similar to other described members of the genus <span class="jp-italic"><a target="xrefwindow" href="https://doi.org/10.1601/nm.9359" title="Dyella - Click to open Names for Life widget" rel="namesforlife-name" class="namesforlife">Dyella</a></span>, but the presence of phosphatidylmethylethanolamine differentiated DHOB07<sup xmlns:f="http://www.example.org/functions">T</sup> from its closest relatives <span class="jp-italic"><a target="xrefwindow" href="https://doi.org/10.1601/nm.27548" title="D. jejuensis - Click to open Names for Life widget" rel="namesforlife-name" class="namesforlife">D. jejuensis</a></span> JP1<sup xmlns:f="http://www.example.org/functions">T</sup>, <span class="jp-italic"><a target="xrefwindow" href="https://doi.org/10.1601/nm.29735" title="D. flagellata - Click to open Names for Life widget" rel="namesforlife-name" class="namesforlife">D. flagellata</a></span> 4M-K16<sup xmlns:f="http://www.example.org/functions">T</sup> and <span class="jp-italic"><a target="xrefwindow" href="https://doi.org/10.1601/nm.29736" title="D. nitratireducens - Click to open Names for Life widget" rel="namesforlife-name" class="namesforlife">D. nitratireducens</a></span> DHG59<sup xmlns:f="http://www.example.org/functions">T</sup>.</p>
<!--splitRids is T2--><!--currentRid is T2--><!--current position is 44--><!--check position is 44--><div class="table-caption-container js-table-caption-container" data-webId="/content/ijsem/10.1099/ijsem.0.002020.T2" id="T2"><h5 class="table-label">Table 2.</h5><p><span>Cellular fatty acid profiles of strain DHOB07<sup xmlns:f="http://www.example.org/functions">T</sup> and type strains of related species of the genus <span class="jp-italic"><a target="xrefwindow" href="https://doi.org/10.1601/nm.9359">Dyella</a></span></span>
<span>Strains: 1, DHOB07<sup xmlns:f="http://www.example.org/functions">T</sup>; 2, <span class="jp-italic"><a target="xrefwindow" href="https://doi.org/10.1601/nm.27548" title="D. jejuensis - Click to open Names for Life widget" rel="namesforlife-name" class="namesforlife">D. jejuensis</a></span> JP1<sup xmlns:f="http://www.example.org/functions">T</sup>; 3, <span class="jp-italic"><a target="xrefwindow" href="https://doi.org/10.1601/nm.29735" title="D. flagellata - Click to open Names for Life widget" rel="namesforlife-name" class="namesforlife">D. flagellata</a></span> 4M-K16<sup xmlns:f="http://www.example.org/functions">T</sup>; 4, <span class="jp-italic"><a target="xrefwindow" href="https://doi.org/10.1601/nm.29736" title="D. nitratireducens - Click to open Names for Life widget" rel="namesforlife-name" class="namesforlife">D. nitratireducens</a></span> DHG59<sup xmlns:f="http://www.example.org/functions">T</sup>; 5, <span class="jp-italic"><a target="xrefwindow" href="https://doi.org/10.1601/nm.29165" title="D. humi - Click to open Names for Life widget" rel="namesforlife-name" class="namesforlife">D. humi</a></span> DHG40<sup xmlns:f="http://www.example.org/functions">T</sup>; 6, <span class="jp-italic"><a target="xrefwindow" href="https://doi.org/10.1601/nm.9534" title="D. koreensis - Click to open Names for Life widget" rel="namesforlife-name" class="namesforlife">D. koreensis</a></span> BB4<sup xmlns:f="http://www.example.org/functions">T</sup>; 7, <span class="jp-italic"><a target="xrefwindow" href="https://doi.org/10.1601/nm.14370" title="D. marensis - Click to open Names for Life widget" rel="namesforlife-name" class="namesforlife">D. marensis</a></span> CS5-B2<sup xmlns:f="http://www.example.org/functions">T</sup> 8, <span class="jp-italic"><a target="xrefwindow" href="https://doi.org/10.1601/nm.29734" title="D. acidisoli - Click to open Names for Life widget" rel="namesforlife-name" class="namesforlife">D. acidisoli</a></span> 4M-Z03<sup xmlns:f="http://www.example.org/functions">T</sup>. Data for strains 1 and 2 from this study, data for strains 3–8 from Chen <span class="jp-italic">et al</span>. [<span class="xref"><a xmlns:f="http://www.example.org/functions" href="#R11">11</a></span>]. All strains were grown on R2A agar at 28 °C for 2 days. Major fatty acids (&gt;10 %) for each strain are indicated in bold. Values are percentages of total fatty acids. –, Not detected or &lt;1 % of the total fatty acid content.</span>
</p></div><p>In conclusion, the major characteristics of strain DHOB07<sup xmlns:f="http://www.example.org/functions">T</sup> were in line with those of the genus <span class="jp-italic"><a target="xrefwindow" href="https://doi.org/10.1601/nm.9359" title="Dyella - Click to open Names for Life widget" rel="namesforlife-name" class="namesforlife">Dyella</a></span>, but the novel strain differed from all described species of the genus by some genotypic, biochemical and chemotaxonomic properties. Therefore, we conclude that strain DHOB07<sup xmlns:f="http://www.example.org/functions">T</sup> represents a novel species of the genus <span class="jp-italic"><a target="xrefwindow" href="https://doi.org/10.1601/nm.9359" title="Dyella - Click to open Names for Life widget" rel="namesforlife-name" class="namesforlife">Dyella</a></span>, for which the name <span class="jp-italic">Dyella lipolytica</span> sp. nov. is proposed.</p>
<div class="clearer"> </div></div><div xmlns:xs="http://www.w3.org/2001/XMLSchema" class="articleSection"><div class="articleSection"><div class="sectionDivider"><div class="tl-main-part title"><a name="s1-18" id="s1-18">Description of <span class="jp-italic">Dyella lipolytica</span> sp. nov.</a></div><div class="menuButton">Go to section...</div><div class="clearer"> </div></div><div class="dropDownMenu"><ul><li><a class="menuLink top-section-link" href="#top-1">TOP</a></li><li><a class="menuLink abstract-section-link" href="#abstract-1">ABSTRACT</a></li><li><a class="menuLink introduction-section-link" href="#introduction-1">INTRODUCTION</a></li><li><a class="menuLink body-section-link" href="#s1-18">Description of <span class="jp-italic">Dyella lipolytica</span> sp. nov.</a></li><li><a class="menuLink ack-section-link" href="#ack1">Funding information</a></li><li><a class="menuLink ack-section-link" href="#ack2">Conflicts of interest</a></li><li><a class="menuLink ref-section-link" href="#references-1">References</a></li></ul></div></div><p><span class="jp-italic"><a target="xrefwindow" href="https://doi.org/10.1601/nm.9359" title="Dyella - Click to open Names for Life widget" rel="namesforlife-name" class="namesforlife">Dyella</a> lipolytica</span> (li.po.ly′ti.ca. Gr. n. <span class="jp-italic">lipos</span> fat; Gr. adj. <span class="jp-italic">lytikos</span> dissolving; N.L. fem. adj. <span class="jp-italic">lipolytica</span> dissolving fat or lipid).</p>
<p>Cells are Gram-stain-negative, aerobic, non-motile, and rod-shaped (0.3–0.6×1.0–2.5 µm). Colonies are circular (1.2–1.7 mm in diameter), convex with clear margin, and yellow after 2 days on R2A agar. Growth is not observed on NA, TSA, LB and MacConkey agar media. Growth on R2A occurs at 10–37 °C, pH 4–7 and in the presence of 0–0.5 % (w/v) NaCl. Optimum growth occurs at 28 °C, pH 5–5.5 and without NaCl supplement. Catalase-positive and oxidase-negative. Nitrate is not reduced to nitrite. Tween 80 are hydrolysed but casein, gelatin, aesculin and starch are not. H<span class="jp-sub">2</span>S and indole are not produced. The following substrates can be used as sole carbon sources for growth: adipic acid, aesculin ferric citrate, arbutin, <span class="jp-small">d</span>-arabinose, <span class="jp-small">d</span>-glucose, melezitose, raffinose, <span class="jp-small">d</span>-sorbitol, <span class="jp-small">d</span>-tagatose, trehalose, erythritol, glycogen, inositol, inulin, <span class="jp-small">l</span>-fucose, malic acid, and <span class="jp-italic">N</span>-acetyl-<span class="jp-small">d</span>-glucosamine. The following enzyme activities are positive: acid phosphatase, cysteine arylamidase, esterase (C4), esterase (C8), <span class="jp-italic">α</span>-fucosidase, <span class="jp-italic">β</span>-galactosidase, <span class="jp-italic">α</span>-glucosidase, leucine arylamidase, <span class="jp-italic">α</span>-mannosidase, <span class="jp-italic">N</span>-acetyl-<span class="jp-italic">β</span>- glucosaminidase and valine arylamidase. he following enzyme activities are negative: alkaline phosphatase, arginine dihydrolase, <span class="jp-italic">α</span>-chymotrypsin, <span class="jp-italic">α</span>-galactosidase, glucose fermentation, <span class="jp-italic">β</span>-glucosidase, <span class="jp-italic">β</span>-glucuronidase, lipase (C14), naphthol-AS-BI-phosphohydrolase, trypsin and urease. Ubiquinone-8 is the respiratory quinone, and iso-C<span class="jp-sub">15 : 0</span>, iso-C<span class="jp-sub">16 : 0</span> and iso-C<span class="jp-sub">17 : 1</span>ω9<span class="jp-italic">c</span> are the major cellular fatty acids (&gt;10 %). The major polar lipids consist of phosphatidylethanolamine, phosphatidylglycerol, diphosphatidylglycerol, phosphatidylmethylethanolamine and several unidentified aminophospholipids and phospholipids.</p>
<p>The type strain DHOB07<sup xmlns:f="http://www.example.org/functions">T</sup> (=NBRC 111473<sup xmlns:f="http://www.example.org/functions">T</sup>=KCTC 52132<sup xmlns:f="http://www.example.org/functions">T</sup>) was isolated from forest soil of Dinghushan Biosphere Reserve, Guangdong Province, PR China. The G+C content of the genomic DNA of the type strain is 59.4 mol%.</p>
<div class="clearer"> </div></div>
</div>

<div id="article-level-0-back">
<div xmlns:xs="http://www.w3.org/2001/XMLSchema" class="articleSection"><div class="articleSection"><div class="sectionDivider"><div class="tl-main-part title"><a name="ack1" id="ack1">Funding information</a></div><div class="menuButton">Go to section...</div><div class="clearer"> </div></div><div class="dropDownMenu"><ul><li><a class="menuLink top-section-link" href="#top-1">TOP</a></li><li><a class="menuLink abstract-section-link" href="#abstract-1">ABSTRACT</a></li><li><a class="menuLink introduction-section-link" href="#introduction-1">INTRODUCTION</a></li><li><a class="menuLink body-section-link" href="#s1-18">Description of <span class="jp-italic">Dyella lipolytica</span> sp. nov.</a></li><li><a class="menuLink ack-section-link" href="#ack1">Funding information</a></li><li><a class="menuLink ack-section-link" href="#ack2">Conflicts of interest</a></li><li><a class="menuLink ref-section-link" href="#references-1">References</a></li></ul></div></div><p>This research was supported by the National Natural Science Foundation of China (grant no. J1310025) and the ‘Climbing’ Program of Guangdong Province (grant no. pdjh2016b0013).</p>
<div class="clearer"> </div></div><div xmlns:xs="http://www.w3.org/2001/XMLSchema" class="articleSection"><div class="articleSection"><div class="sectionDivider"><div class="tl-main-part title"><a name="ack2" id="ack2">Conflicts of interest</a></div><div class="menuButton">Go to section...</div><div class="clearer"> </div></div><div class="dropDownMenu"><ul><li><a class="menuLink top-section-link" href="#top-1">TOP</a></li><li><a class="menuLink abstract-section-link" href="#abstract-1">ABSTRACT</a></li><li><a class="menuLink introduction-section-link" href="#introduction-1">INTRODUCTION</a></li><li><a class="menuLink body-section-link" href="#s1-18">Description of <span class="jp-italic">Dyella lipolytica</span> sp. nov.</a></li><li><a class="menuLink ack-section-link" href="#ack1">Funding information</a></li><li><a class="menuLink ack-section-link" href="#ack2">Conflicts of interest</a></li><li><a class="menuLink ref-section-link" href="#references-1">References</a></li></ul></div></div><p>The authors declare that there are no conflicts of interest.</p>
<div class="clearer"> </div></div><div xmlns:xs="http://www.w3.org/2001/XMLSchema" class="articleSection"><div class="articleSection"><div class="sectionDivider"><div class="tl-main-part title"><a name="references-1" id="references-1">References</a></div><div class="menuButton">Go to section...</div><div class="clearer"> </div></div><div class="dropDownMenu"><ul><li><a class="menuLink top-section-link" href="#top-1">TOP</a></li><li><a class="menuLink abstract-section-link" href="#abstract-1">ABSTRACT</a></li><li><a class="menuLink introduction-section-link" href="#introduction-1">INTRODUCTION</a></li><li><a class="menuLink body-section-link" href="#s1-18">Description of <span class="jp-italic">Dyella lipolytica</span> sp. nov.</a></li><li><a class="menuLink ack-section-link" href="#ack1">Funding information</a></li><li><a class="menuLink ack-section-link" href="#ack2">Conflicts of interest</a></li><li><a class="menuLink ref-section-link" href="#references-1">References</a></li></ul></div></div><span class="references"><ol class="references"><li><span class="reference" id="R1">Xie CH, Yokota A. <span class="jp-italic">Dyella japonica</span> gen. nov., sp. nov., a γ-proteobacterium isolated from soil. <span class="reference-link-start"> </span>Int J Syst Evol Microbiol. 2005;55:753–6.<span class="reference-link-end"> </span> doi: <a href="http://dx.doi.org/10.1099/ijs.0.63377-0">http://dx.doi.org/10.1099/ijs.0.63377-0</a>. PubMed PMID: 15774657.</span></li><li><span class="reference" id="R2">An DS, Im WT, Yang HC, Yang DC, Lee ST. <span class="jp-italic">Dyella koreensis</span> sp. nov., a β-glucosidase-producing bacterium. <span class="reference-link-start"> </span>Int J Syst Evol Microbiol. 2005;55:1625–8.<span class="reference-link-end"> </span> doi: <a href="http://dx.doi.org/10.1099/ijs.0.63695-0">http://dx.doi.org/10.1099/ijs.0.63695-0</a>. PubMed PMID: 16014492.</span></li><li><span class="reference" id="R3">Son HM, Yang JE, Yi EJ, Park Y, Won KH, et al. <span class="jp-italic">Dyella kyungheensis</span> sp. nov., isolated from soil of a cornus fruit field. <span class="reference-link-start"> </span>Int J Syst Evol Microbiol. 2013;63:3807–11.<span class="reference-link-end"> </span> doi: <a href="http://dx.doi.org/10.1099/ijs.0.051185-0">http://dx.doi.org/10.1099/ijs.0.051185-0</a>. PubMed PMID: 23667147.</span></li><li><span class="reference" id="R4">Jung HM, Ten LN, Kim KH, An DS, Im WT, et al. <span class="jp-italic">Dyella ginsengisoli</span> sp. nov., isolated from soil of a ginseng field in South Korea. <span class="reference-link-start"> </span>Int J Syst Evol Microbiol. 2009;59:460–5.<span class="reference-link-end"> </span> doi: <a href="http://dx.doi.org/10.1099/ijs.0.64514-0">http://dx.doi.org/10.1099/ijs.0.64514-0</a>. PubMed PMID: 19244422.</span></li><li><span class="reference" id="R5">Zhao F, Guo XQ, Wang P, He LY, Huang Z, et al. <span class="jp-italic">Dyella jiangningensis</span> sp. nov., a γ-proteobacterium isolated from the surface of potassium-bearing rock. <span class="reference-link-start"> </span>Int J Syst Evol Microbiol. 2013;63:3154–7.<span class="reference-link-end"> </span> doi: <a href="http://dx.doi.org/10.1099/ijs.0.048470-0">http://dx.doi.org/10.1099/ijs.0.048470-0</a>. PubMed PMID: 23435246.</span></li><li><span class="reference" id="R6">Lee DW, Lee SD. <span class="jp-italic">Dyella marensis</span> sp. nov., isolated from cliff soil. <span class="reference-link-start"> </span>Int J Syst Evol Microbiol. 2009;59:1397–400.<span class="reference-link-end"> </span> doi: <a href="http://dx.doi.org/10.1099/ijs.0.000968-0">http://dx.doi.org/10.1099/ijs.0.000968-0</a>. PubMed PMID: 19502323.</span></li><li><span class="reference" id="R7">Weon HY, Anandham R, Kim BY, Hong SB, Jeon YA, et al. <span class="jp-italic">Dyella soli</span> sp. nov. and <span class="jp-italic">Dyella terrae</span> sp. nov., isolated from soil. <span class="reference-link-start"> </span>Int J Syst Evol Microbiol. 2009;59:1685–90.<span class="reference-link-end"> </span> doi: <a href="http://dx.doi.org/10.1099/ijs.0.004838-0">http://dx.doi.org/10.1099/ijs.0.004838-0</a>. PubMed PMID: 19542132.</span></li><li><span class="reference" id="R8">Anandham R, Kwon SW, Indira Gandhi P, Kim SJ, Weon HY, et al. <span class="jp-italic">Dyella thiooxydans</span> sp. nov., a facultatively chemolithotrophic, thiosulfate-oxidizing bacterium isolated from rhizosphere soil of sunflower (<span class="jp-italic">Helianthus annuus</span> L.). <span class="reference-link-start"> </span>Int J Syst Evol Microbiol. 2011;61:392–8.<span class="reference-link-end"> </span> doi: <a href="http://dx.doi.org/10.1099/ijs.0.022012-0">http://dx.doi.org/10.1099/ijs.0.022012-0</a>. PubMed PMID: 20305058.</span></li><li><span class="reference" id="R9">Kim MS, Hyun DW, Kim JY, Kim S, Bae JW, et al. <span class="jp-italic">Dyella jejuensis</span> sp. nov., isolated from soil of Hallasan Mountain in Jeju Island. <span class="reference-link-start"> </span>J Microbiol. 2014;52:373–7.<span class="reference-link-end"> </span> doi: <a href="http://dx.doi.org/10.1007/s12275-014-3670-7">http://dx.doi.org/10.1007/s12275-014-3670-7</a>. PubMed PMID: 24810316.</span></li><li><span class="reference" id="R10">Chen MH, Lv YY, Wang J, Tang L, Qiu LH. <span class="jp-italic">Dyella humi</span> sp. nov., isolated from forest soil. <span class="reference-link-start"> </span>Int J Syst Evol Microbiol. 2016;66:4372–6.<span class="reference-link-end"> </span> doi: <a href="http://dx.doi.org/10.1099/ijsem.0.001358">http://dx.doi.org/10.1099/ijsem.0.001358</a>. PubMed PMID: 27487774.</span></li><li><span class="reference" id="R11">Chen MH, Xia F, Lv YY, Zhou XY, Qiu LH. <span class="jp-italic">Dyella acidisoli</span> sp. nov., <span class="jp-italic">Dyella flagellata</span> sp. nov. and <span class="jp-italic">Dyella nitratireducens</span> sp. nov., isolated from forest soil. <span class="reference-link-start"> </span>Int J Syst Evol Microbiol. 2017;67:736–43.<span class="reference-link-end"> </span> doi: <a href="http://dx.doi.org/10.1099/ijsem.0.001716">http://dx.doi.org/10.1099/ijsem.0.001716</a>. PubMed PMID: 27930272.</span></li><li><span class="reference" id="R12">Gerhardt P, Murray RGE, Wood WA, Krieg NR. <span class="reference-link-start"> </span><span class="reference-link-start"> </span>Methods for General and Molecular Bacteriology. Washington, DC: American Society for Microbiology; 1994. </span></li><li><span class="reference" id="R13">Xu P, Li WJ, Tang SK, Zhang YQ, Chen GZ, et al. <span class="jp-italic">Naxibacter alkalitolerans</span> gen. nov., sp. nov., a novel member of the family '<span class="jp-italic">Oxalobacteraceae</span>' isolated from China. <span class="reference-link-start"> </span>Int J Syst Evol Microbiol. 2005;55:1149–53.<span class="reference-link-end"> </span> doi: <a href="http://dx.doi.org/10.1099/ijs.0.63407-0">http://dx.doi.org/10.1099/ijs.0.63407-0</a>. PubMed PMID: 15879247.</span></li><li><span class="reference" id="R14">Brown AE. <span class="reference-link-start"> </span><span class="reference-link-start"> </span><span class="jp-italic">Benson</span>’<span class="jp-italic">s Microbiological Applications: Laboratory Manual in General Microbiology</span>. 4th ed. New York: McGraw-Hill; 1985. </span></li><li><span class="reference" id="R15">Atlas RM. Composition of media. In: Parks LC, editor. <span class="reference-link-start"> </span><span class="reference-link-start"> </span>Handbook of Microbiology Media. 2nd ed. Boca Raton, FL: CRC Press; 1993.  In (editor)</span></li><li><span class="reference" id="R16">Bauer AW, Kirby WM, Sherris JC, Turck M. Antibiotic susceptibility testing by a standardized single disk method. <span class="reference-link-start"> </span>Am J Clin Pathol. 1966;36:49.</span></li><li><span class="reference" id="R17">Saitou N, Nei M. The neighbor-joining method: a new method for reconstructing phylogenetic trees. <span class="reference-link-start"> </span>Mol Biol Evol. 1987;4:406–25.<span class="reference-link-end"> </span> PubMed PMID: 3447015.</span></li><li><span class="reference" id="R18">Fitch WM. Toward defining the course of evolution: minimum change for a specific tree Topology. <span class="reference-link-start"> </span>Syst Biol. 1971;20:406–16.<span class="reference-link-end"> </span> doi: <a href="http://dx.doi.org/10.1093/sysbio/20.4.406">http://dx.doi.org/10.1093/sysbio/20.4.406</a>.</span></li><li><span class="reference" id="R19">Felsenstein J. Evolutionary trees from DNA sequences: a maximum likelihood approach. <span class="reference-link-start"> </span>J Mol Evol. 1981;17:368–76.<span class="reference-link-end"> </span> doi: <a href="http://dx.doi.org/10.1007/BF01734359">http://dx.doi.org/10.1007/BF01734359</a>. PubMed PMID: 7288891.</span></li><li><span class="reference" id="R20">Tamura K, Peterson D, Peterson N, Stecher G, Nei M, et al. MEGA5: molecular evolutionary genetics analysis using maximum likelihood, evolutionary distance, and maximum parsimony methods. <span class="reference-link-start"> </span>Mol Biol Evol. 2011;28:2731–9.<span class="reference-link-end"> </span> doi: <a href="http://dx.doi.org/10.1093/molbev/msr121">http://dx.doi.org/10.1093/molbev/msr121</a>. PubMed PMID: 21546353.</span></li><li><span class="reference" id="R21">Kimura M. A simple method for estimating evolutionary rates of base substitutions through comparative studies of nucleotide sequences. <span class="reference-link-start"> </span>J Mol Evol. 1980;16:111–20.<span class="reference-link-end"> </span> doi: <a href="http://dx.doi.org/10.1007/BF01731581">http://dx.doi.org/10.1007/BF01731581</a>. PubMed PMID: 7463489.</span></li><li><span class="reference" id="R22">Felsenstein J. Confidence limits on phylogenies: an approach using the bootstrap. <span class="reference-link-start"> </span>Evolution. 1985;39:783–91.<span class="reference-link-end"> </span> doi: <a href="http://dx.doi.org/10.2307/2408678">http://dx.doi.org/10.2307/2408678</a>.</span></li><li><span class="reference" id="R23">Mesbah M, Premachandran U, Whitman WB. Precise measurement of the G+C content of deoxyribonucleic acid by high-performance liquid chromatography. <span class="reference-link-start"> </span>Int J Syst Bacteriol. 1989;39:159–67.<span class="reference-link-end"> </span> doi: <a href="http://dx.doi.org/10.1099/00207713-39-2-159">http://dx.doi.org/10.1099/00207713-39-2-159</a>.</span></li><li><span class="reference" id="R24">Marmur J, Doty P. Determination of the base composition of deoxyribonucleic acid from its thermal denaturation temperature. <span class="reference-link-start"> </span>J Mol Biol. 1962;5:109–18.<span class="reference-link-end"> </span> doi: <a href="http://dx.doi.org/10.1016/S0022-2836(62)80066-7">http://dx.doi.org/10.1016/S0022-2836(62)80066-7</a>. PubMed PMID: 14470099.</span></li><li><span class="reference" id="R25">De Ley J, Cattoir H, Reynaerts A. The quantitative measurement of DNA hybridization from renaturation rates. <span class="reference-link-start"> </span>Eur J Biochem. 1970;12:133–42.<span class="reference-link-end"> </span> doi: <a href="http://dx.doi.org/10.1111/j.1432-1033.1970.tb00830.x">http://dx.doi.org/10.1111/j.1432-1033.1970.tb00830.x</a>. PubMed PMID: 4984993.</span></li><li><span class="reference" id="R26">Wayne LG, Brenner D, Colwell R, Grimont P, Kandler O, et al. International Committee on Systematic Bacteriology. Report of the ad hoc committee on reconciliation of approaches to bacterial systematics. <span class="reference-link-start"> </span>Int J Syst Bacteriol. 1987;37:463–4.<span class="reference-link-end"> </span> doi: <a href="http://dx.doi.org/10.1099/00207713-37-4-463">http://dx.doi.org/10.1099/00207713-37-4-463</a>.</span></li><li><span class="reference" id="R27">Kuykendall LD, Roy MA, O'Neill JJ, Devine TE. Fatty acids, antibiotic resistance, and deoxyribonucleic acid homology groups of <span class="jp-italic">Bradyrhizobium japonicum</span>. <span class="reference-link-start"> </span>Int J Syst Bacteriol. 1988;38:358–61.<span class="reference-link-end"> </span> doi: <a href="http://dx.doi.org/10.1099/00207713-38-4-358">http://dx.doi.org/10.1099/00207713-38-4-358</a>.</span></li><li><span class="reference" id="R28">Minnikin DE, O'Donnell AG, Goodfellow M, Alderson G, Athalye M, et al. An integrated procedure for the extraction of bacterial isoprenoid quinones and polar lipids. <span class="reference-link-start"> </span>J Microbiol Methods. 1984;2:233–41.<span class="reference-link-end"> </span> doi: <a href="http://dx.doi.org/10.1016/0167-7012(84)90018-6">http://dx.doi.org/10.1016/0167-7012(84)90018-6</a>.</span></li><li><span class="reference" id="R29">Kroppenstedt RM. Separation of bacterial menaquinones by HPLC using reverse phase (RP18) and a silver loaded ion exchanger as stationary phases. <span class="reference-link-start"> </span>J Liq Chromatogr. 1982;5:2359–67.<span class="reference-link-end"> </span> doi: <a href="http://dx.doi.org/10.1080/01483918208067640">http://dx.doi.org/10.1080/01483918208067640</a>.</span></li></ol></span><div class="clearer"> </div></div><div xmlns:f="http://www.example.org/functions" class="copyright-after-text">© 2017 IUMS</div>
</div>

<div id="article-level-0-figs-and-tables" class="bm">

</div>

<div id="article-level-0-end-metadata" class="fm">

</div></div>